### ISSUE TITLE
Yarn package potential issue: Duplicate vue.js libraries in yarn.lock

### ISSUE THREAD

**Issue by rpocklin** [owncloud/web#4279]
When checking out the repo in master, the current `yarn.lock` has the following:
```
vue@^2.6.10:
  version "2.6.11"
  resolved "https://registry.yarnpkg.com/vue/-/vue-2.6.11.tgz#76594d877d4b12234406e84e35275c6d514125c5"
  integrity sha512-VfPwgcGABbGAue9+sfrD4PuwFar7gPb1yl1UK1MwXoQPAw0BKSqWfoYCT/ThFrdEVWoI51dBuyCoiNU9bZDZxQ==

vue@^2.6.11:
  version "2.6.12"
  resolved "https://registry.yarnpkg.com/vue/-/vue-2.6.12.tgz#f5ebd4fa6bd2869403e29a896aed4904456c9123"
  integrity sha512-uhmLFETqPPNyuLLbsKz6ioJ4q7AZHzD8ZVFNATNyICSZouqP2Sz0rotWQC8UNBF6VGSCs5abnKJoStA6JbCbfg==
```

I believe this is because of the recent update to `owncloud-design-system@^1.14.0` which specifies ` vue "^2.6.11"` as a dependency.  Generally it is bad to have multiple versions of a framework in builds as this can lead to duplicate code or subtle errors between version revisions (though unlikely if **vue** is following semver correctly).

I can create a PR for what I believe is a correct `yarn.lock` for this situation.

**Comment by pascalwengerter** [comment]
@rpocklin can this issue safely be closed after your PR has been merged? 🙌🏽

**Comment by pascalwengerter** [comment]
Closed via #4280
